deploy/kubernetes/docker/build.sh

```diff
@@ -51,6 +54,10 @@ while (( "$#" )); do
     --help)
       exit_with_usage
       ;;
+    --apache-mirror)
+      APACHE_MIRROR="$2"
```

Comment:
Perhaps we can decide not to replace the default value if the value passed in is empty?

Reply:
Ok